src/dash-table/components/CellLabel/index.tsx

```diff
@@ -15,11 +16,34 @@ export default class CellLabel extends PureComponent<IProps> {
         } = this.props;
 
         return (<div
+            ref='el'
             className={className}
+            tabIndex={-1}
```

Comment:
this makes the element focusable

Reply:
neat, I didn't know about that
https://webaim.org/techniques/keyboard/tabindex

> A tabindex="-1" value removes the element from the default navigation flow and also allows it to receive programmatic focus.

Seems like the right call (as opposed to 0, keeping it in the tab "navigation flow")